node/strings.go

```diff
@@ -2,7 +2,8 @@ package node
 
 import "strings"
 
-//move from string.go. filter out empty string too.
+// SplitAndTrimEmpty is copied from common/string.go
```

Comment:
Whats the point of saying that it is copied from `common/string.go`, we can just delete it from there as its only used here.